vita3k/gui/src/game_context_menu.cpp

```diff
@@ -220,7 +220,8 @@ void game_context_menu(GuiState &gui, HostState &host) {
         ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
         ImGui::SetNextWindowSize(display_size, ImGuiCond_Always);
         ImGui::Begin("##context_dialog", nullptr, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoBackground | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoSavedSettings);
-        ImGui::SetNextWindowPosCenter();
+        ImGui::SetNextWindowBgAlpha(0.999f);
```

Comment:
what is background alpha 0.999 going to do? won't it look the same as 100%?

Reply:
0.999 is 100 % 1.0 get broken result